docs/integrations/ios-specific-workflow-steps/swift-lint.md

```diff
@@ -0,0 +1,38 @@
+---
+title: SwiftLint
+metaTitle: SwiftLint
+metaDescription: SwiftLint
+sidebar_position: 4
+---
+
+import Screenshot from '@site/src/components/Screenshot';
+
+# SwiftLint
+This step installs [SwiftLint](https://github.com/realm/SwiftLint/) and runs swiftlint with given options. Note that if you are using Cocoapods, you should add this step after the **Cocoapods Install** step. 
+
+<Screenshot url='https://cdn.appcircle.io/docs/assets/BE2613-lint_order.png' />
+
+:::caution
+If you are using **Swift Package Manager (SPM)**, do not use this step. SPM packages will be compiled in other steps that work with **Xcode**, such as **Xcodebuild for Devices.**
+:::
+:::warning
+If you are using **Cocoapods**, note that this step is dependent on the **Cocoapods Install** step. Otherwise the SwiftLint component will **fail** and the **pipeline will break.**
+:::
+
+You can also customize your SwiftLint step with the options in the component.
```

Comment:
I added both input and output variable tables at the end of doc pages, but I did not delete screen shots because it has more details, and specifies some specific areas.